transport/can/can-buffer.c

```diff
@@ -23,12 +23,14 @@
 
 void AsebaSendBuffer(AsebaVMState *vm, const uint8* data, uint16 length)
 {
+	(void) vm;
```

Comment:
You are correct: this prevents warnings about unused variables. I do not have a "serious" reference available but I have seen it used on several open source projects. See also this SO answer: http://stackoverflow.com/a/308286